doc/whats_new.rst

```diff
@@ -100,6 +100,10 @@ Enhancements
     - Extensive plotting example shows how to use contours and filled contours
       on glass brain.
 
+    - Function :func:`nilearn._utils.ndimage.largest_connected_component_img`
```

Comment:
>did this break the check ?

CircleCI build failed because reference you have given as ```nilearn._utils.ndimage.largest_connected_component_img``` could not be found. Since this function is not initialized in ```nilearn/_utils/__init__.py``` from ndimage module.

We could initialize this function in ```_utils/__init__.py``` from ndimage and refer as ``nilearn._utils.largest_connected_component_img``` directly in whats_new.rst. In this way we can find the target and build will go through.